pkg/kubelet/apis/podresources/server_v1_test.go

```diff
@@ -891,7 +890,6 @@ func TestGetPodResourcesV1(t *testing.T) {
 	pluginCDIDevices := []*podresourcesapi.CDIDevice{{Name: "dra-dev0"}, {Name: "dra-dev1"}}
 	draDevs := []*podresourcesapi.DynamicResource{
 		{
-			ClassName:      "resource-class",
 			ClaimName:      "claim-name",
 			ClaimNamespace: "default",
 			ClaimResources: []*podresourcesapi.ClaimResource{{CDIDevices: pluginCDIDevices}},
```

Comment:
Should we expand these tests to include tests for the the additional fields in ClaimResources now?

Reply:
Contributions welcome...

I also need to write more validation tests.

Reply:
I've added the new fields. But I am not sure how useful these tests are. They don't check that a real server produces correct data, only that `v1PodResourcesServer.getContainerResources` really calls through to the (mock) provider.

Reply:
I think it's fine for now. We need more comprehensive tests here in general
/cc @ffromani

Reply:
I agree: fine for now, we need to invest into more testing in the future (I can help here)

Reply:
I created https://github.com/kubernetes/kubernetes/issues/126268 and added it to the DRA backlog.